panic in MinerSectorEvents Extractor

### Describe the bug:
```
{"level":"info","ts":"2022-09-01T15:46:25.147Z","logger":"lily/index/manager","caller":"integrated/manager.go:125","msg":"task success","height":"2123465","reporter":"analysis-worker","task":"actor","status":"OK","duration":5.480898685}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x209b11f]

goroutine 1473596 [running]:
github.com/filecoin-project/lily/tasks/actorstate/miner.ExtractMinerSectorStateEvents({0x4ee2e68, 0xd4196e2d80}, 0x0)
	/go/src/github.com/filecoin-project/lily/tasks/actorstate/miner/sector_events.go:133 +0x7f
github.com/filecoin-project/lily/tasks/actorstate/miner.ExtractSectorEvents({0x4ee2e68, 0xd4196e2d80}, 0xcd23604ae0, 0x26, 0x0)
	/go/src/github.com/filecoin-project/lily/tasks/actorstate/miner/sector_events.go:110 +0x4b
github.com/filecoin-project/lily/tasks/actorstate/miner.SectorEventsExtractor.Extract({}, {0x4eaf360, 0xc74953b3b0}, {{{{0xcd23604a80, 0x26}}, {{0xcd23604ae0, 0x26}}, 0x0, {0xcb0b393620}}, 0x1, ...}, ...)
	/go/src/github.com/filecoin-project/lily/tasks/actorstate/miner/sector_events.go:100 +0xd54
github.com/filecoin-project/lily/tasks/actorstate.(*Task).startActorStateExtraction.func1.1()
	/go/src/github.com/filecoin-project/lily/tasks/actorstate/task.go:156 +0x226
created by github.com/filecoin-project/lily/tasks/actorstate.(*Task).startActorStateExtraction.func1
	/go/src/github.com/filecoin-project/lily/tasks/actorstate/task.go:146 +0x545
```

### Steps to Reproduce:
extract miner sector events on mainnet at height `2123465`

### Lily Version: v0.12.0-rc1

--- a/tasks/actorstate/miner/sector_events.go
+++ b/tasks/actorstate/miner/sector_events.go
@@ -34,9 +34,14 @@ func (SectorEventsExtractor) Extract(ctx context.Context, a actorstate.ActorInfo
 	}
 
 	var (
-		sectorChanges      *miner.SectorChanges
-		preCommitChanges   *miner.PreCommitChanges
-		sectorStateChanges *SectorStateEvents
+		sectorChanges      = miner.MakeSectorChanges()
+		preCommitChanges   = miner.MakePreCommitChanges()
+		sectorStateChanges = &SectorStateEvents{
+			Removed:    bitfield.New(),
+			Recovering: bitfield.New(),
+			Faulted:    bitfield.New(),
+			Recovered:  bitfield.New(),
+		}
 	)
 	if extState.ParentState() == nil {
 		// If the miner doesn't have previous state list all of its current sectors and precommits
@@ -45,12 +50,10 @@ func (SectorEventsExtractor) Extract(ctx context.Context, a actorstate.ActorInfo
 			return nil, fmt.Errorf("loading miner sectors: %w", err)
 		}
 
-		sectorChanges = miner.MakeSectorChanges()
 		for _, sector := range sectors {
 			sectorChanges.Added = append(sectorChanges.Added, *sector)
 		}
 
-		preCommitChanges = miner.MakePreCommitChanges()
 		if err = extState.CurrentState().ForEachPrecommittedSector(func(info minertypes.SectorPreCommitOnChainInfo) error {
 			preCommitChanges.Added = append(preCommitChanges.Added, info)
 			return nil

